codes/c/chapter_array_and_linkedlist/linked_list.c

```diff
@@ -0,0 +1,86 @@
+/**
+ * File: linked_list.c
+ * Created Time: 2022-01-12
+ * Author: Zero (glj0@outlook.com)
+ */
+
+#include "../include/include.h"
+
+ /* 在链表的结点 n0 之后插入结点 P */
+void insertNode(ListNode* n0, ListNode* P) {
```

Comment:
Keep simple, remove "Node"  from function name of "insertNode".

Reply:
It is a problem. Function name in include/ should be Capitalize the first letter, in case the function names in other chapters are verbose.

Reply:
> It is a problem.Only keep "removeNode".

Yes, there will be such problems in c, and such problems will definitely be encountered in subsequent coding. A rule is needed, and coding is done according to the rules